### ISSUE TITLE
Object has no attribute

### ISSUE THREAD

**Issue by VirtualPenman** [lifehackjim/cert_human#3]
Hey there, I am fairly new to Python and trying to connect to a site through a proxy and get certificate information. (I did not know where else to post this so I really apologize if this is not the right place)
Whenever I run any of the request methods to get certs I get an object has not attribute error or an error that pertains to urllib3. 

**To Reproduce**
I am following the examples in the documentation on https://cert-human.readthedocs.io/en/latest/api/api.html

**Expected behavior**
I am trying to follow all the examples to print the cert resultssni

**Screenshots**
![3Capture](https://user-images.githubusercontent.com/10656823/63053822-e2766c00-be9f-11e9-8c80-8e29c913048a.PNG)

![2Capture](https://user-images.githubusercontent.com/10656823/63053828-e6a28980-be9f-11e9-8745-5fd573425d63.PNG)

![1Capture](https://user-images.githubusercontent.com/10656823/63053829-e86c4d00-be9f-11e9-8edf-6faa440f2428.PNG)

![Capture](https://user-images.githubusercontent.com/10656823/63053834-eacea700-be9f-11e9-8cf7-e2bc858e6f27.PNG)

**Desktop (please complete the following information):**
 - OS: [Win 10]
- Python 3.7.4 Shell

The reason I have proxies=proxies is I am behind a proxy at work and must submit a username and password to allow the proxy to access a site. This was the reason I wanted to use requests.get since they have a proxy parameter built in that I can pass. I already have defined that proxy above. I was really excited when I found your build and I assume I am just using it incorrectly. 

Again I did not know what other capacity to reach you directly as I am so new, so I apologize.

**Comment by KyleKing** [comment]
tldr: Use requests `2.22.0` for now

I have the same problem

```py
import cert_human
import requests

with cert_human.urllib3_patch():
    response = requests.get('https://www.google.com')
    store = cert_human.CertStore.from_response(response=response)
```

Throws a missing attribute on `raw.peer`

```sh
Traceback (most recent call last):
  File "get_certs.py", line 21, in <module>
    store = cert_human.CertStore.from_response(response=response)
  File "C:\..\.venv\lib\site-packages\cert_human\__init__.py", line 454, in from_response
    raise CertHumanError(error)
cert_human.CertHumanError: Response missing attribute 'raw.peer_cert', not issued using enable_urllib3_patch      
```

This error is only shown when using requests 2.25, but not requests 2.22 (I have not tested other versions)

**Comment by kolinger** [comment]
This did break since requests==2.24.0 (thus last working version is requests==2.23.0) since pyOpenSSL dependency was removed, see changelog: https://github.com/psf/requests/blob/master/HISTORY.md#2240-2020-06-17

Currently cert_human requires pyOpenSSL in order to work and so if requests don't use pyOpenSSL then cart_human can't work.

But there is workaround how to enable old behaviour where requests use pyOpenSSL for SSL/TLS instead of built in ssl library, see https://github.com/psf/requests/pull/5443#issuecomment-645740394

In result this can be fixed with following line (call before calling requests):
`urllib3.contrib.pyopenssl.inject_into_urllib3()`

With this workaround cert_human works even with current requests version (2.26.0).

**Comment by henribru** [comment]
pyOpenSSL support has been deprecated now and will be removed in urllib3 v2.1.0
https://github.com/urllib3/urllib3/issues/2680
https://urllib3.readthedocs.io/en/2.0.0a2/reference/contrib/pyopenssl.html
